### ISSUE TITLE
Important bug

### ISSUE THREAD

**Issue by sedoLevi609** [radimitrov/CSharpShellApp#388]
@radimitrov  I found a bug: if i type code it sometimes freezes the app if I didn't wait while intellisense shows the keywords it finds using the first letter. Please fix this!

**Comment by sedoLevi609** [comment]
Now I can't reproduce the issue,but yesterday it was a problem

**Comment by radimitrov** [comment]
I did encounter this freeze once a few weeks ago, however I attributed it to the emulator and VS debugger being problematic. Should be easily fixable, though

**Comment by radimitrov** [comment]
Should be fixed in the update today/tomorrow

**Comment by sedoLevi609** [comment]
@radimitrov  it is not fixed

**Comment by radimitrov** [comment]
@sedoLevi609  Unfortunately there isn't much else I can do right now. The problem can't be reproduced and there is no stack trace to work off hence the cause is unknown.
Can this happen at any point when typing? Or just after initially opening a project? Or the app itself?

**Comment by sedoLevi609** [comment]
Maybe if I want to load a keyword from the loaded list when typing,then I choose one,then continue typing and just froze sometimes. The list is loaded after only a few seconds mostly(i'm not always waiting it,just I continue typing,which also could be the cause,but just a guess;it's shown in the provided video),and immedatly after the load it loads not the right words(if I'm typing fast),only if I type in more letters then shows the right keywords(it is also shown in the provided video),what is strange

https://github.com/user-attachments/assets/0729a856-21e4-401d-b84d-4e0e96b190d3

**Comment by radimitrov** [comment]
Yeah, I completely misunderstood the issue. I thought the app was freezing.
I think I replicated the issue here. It shouldn't occur, but there is a way I can fix this. There also appear to be snippet suggestions when there shouldn't be any, so that needs fixing too. The delay of a second or two is when new classes need to be loaded the first time.

https://github.com/user-attachments/assets/fdd18713-3078-4f2f-9ba4-59c8ef721952
